rlclientlib/schema/v2/Event.fbs

```diff
@@ -9,12 +9,15 @@ table Event {
 }
 
 table BatchMetadata {
-    content_encoding: string;
+    content_encoding: string; //valid values: IDENTITY and DEDUP
```

Comment:
should this maybe be an enum?

Reply:
A string allows for open ended values so that unknown values can be properly inspected by intermediaries.